code/obj/item/clothing/armor.dm

```diff
@@ -316,6 +318,22 @@
 		icon_state = "centcom-red"
 		item_state = "centcom-red"
 
+/obj/item/clothing/suit/armor/centcommcoat //coat version of the centcom armour
+	name = "administator's coat"
+	desc = "A luxorious formal coat. It is specifically made for Nanotrasen commanders. It seems to be made out of some thermally resistant material."
+	icon_state = "centcoat"
+	item_state = "centcoat"
+	setupProperties()
+		..()
+		setProperty("coldprot", 30)
+		setProperty("heatprot", 30)
```

Comment:
are these supposed to be lower than on `capcoat`?

Reply:
It is not, good catch yass!